client/components/SignedIn/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const SignedIn = () => {
+    return (
+        <React.Fragment>
```

Comment:
I think the `SignedIn` and `SignedOut` have been wrongly named. Also, Can't these two be just one component ? Then in the Navbar you use conditional checks in the jsx to render what needs to be rendered ?